plugins/mac-vth264/encoder.c

```diff
@@ -145,14 +147,26 @@ static OSStatus session_set_prop(VTCompressionSessionRef session,
 }
 
 static OSStatus session_set_bitrate(VTCompressionSessionRef session,
+				    const char *rate_control, int quality,
 				    int new_bitrate, bool limit_bitrate,
 				    int max_bitrate, float max_bitrate_window)
 {
 	OSStatus code;
 
-	SESSION_CHECK(session_set_prop_int(
-		session, kVTCompressionPropertyKey_AverageBitRate,
-		new_bitrate * 1000));
+	bool rc_constant_quality = astrcmpi(rate_control, "Constant quality") ==
```

Comment:
I'm not really happy that this is "stringly typed" - from what I could tell this is not a user-defined text and as such should either be a numeric value or an ENUM. Using string comparison for this purpose seems wasteful to me.